streamer/src/nonblocking/quic.rs

```diff
@@ -120,6 +120,8 @@ pub async fn run_server(
                 stats.clone(),
             ));
             sleep(Duration::from_micros(WAIT_BETWEEN_NEW_CONNECTIONS_US)).await;
+        } else {
+            debug!("accept(): Timed out waiting for connection");
```

Comment:
In here it is incoming.next() timing out. The accept is a function in the 0.9.3 Quinn library.

Reply:
looks like same semantics, better name.  that is, `Incoming::next()` was basically POSIX `select(2)` anyway.  is it worth using the literal name, which doesn't necessarily imply timeout cause?  i think most developers versed in network programming will recognize "accept()" and likely ignore the line 'cause that's what's supposed to happen.  whereas switching to "Incoming::next()` would probably lead to something like, "wtf?  let's look at this code," only to come to the same conclusion later

Reply:
Fair argument